AmlaMarketPlace.DAL/Data/AmlaMarketPlaceDbContext.cs

```diff
@@ -15,13 +15,52 @@ public AmlaMarketPlaceDbContext(DbContextOptions<AmlaMarketPlaceDbContext> optio
     {
     }
 
+    public virtual DbSet<Image> Images { get; set; }
+
+    public virtual DbSet<Product> Products { get; set; }
+
+    public virtual DbSet<Status> Statuses { get; set; }
+
     public virtual DbSet<User> Users { get; set; }
 
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
-        => optionsBuilder.UseSqlServer("Name=DefaultConnection");
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. You can avoid scaffolding the connection string by using the Name= syntax to read it from configuration - see https://go.microsoft.com/fwlink/?linkid=2131148. For more guidance on storing connection strings, see https://go.microsoft.com/fwlink/?LinkId=723263.
+        => optionsBuilder.UseSqlServer("Server=ACL-117\\SQL2022;Database=AmlaMarketPlaceDB;User Id=sa;Password=Commerce#765;TrustServerCertificate=True;");
```

Comment:
Use Connection string name i.e., "DefaultConnection" instead of writing the actual connection string inside that UseSqlServer. So replace this:

 => optionsBuilder.UseSqlServer("Server=ACL-117\\SQL2022;Database=AmlaMarketPlaceDB;User Id=sa;Password=Commerce#765;TrustServerCertificate=True;"); 
 
 with this:
 
   => optionsBuilder.UseSqlServer("Name=DefaultConnection");